sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -96,6 +96,29 @@ public void waitUntilIdentifiedElementIsVisibleAndClickable(final Object selecto
     }
   }
 
+  public void waitUntilIdentifiedElementDisappear(final Object selector) {
+    waitUntilIdentifiedElementDisappear(selector, FLUENT_WAIT_TIMEOUT_SECONDS);
+  }
+
+  public void waitUntilIdentifiedElementDisappear(final Object selector, int seconds) {
+    if (selector instanceof By) {
+      assertHelpers.assertWithPoll(
+          () -> {
+            assertThat(baseSteps.getDriver().findElement((By) selector).isEnabled()).isFalse();
+            assertThat(baseSteps.getDriver().findElement((By) selector).isDisplayed()).isFalse();
```

Comment:
Please add messages to have a clear description on assertion error

Reply:
done